lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -1158,7 +904,7 @@ def eb():
 
         return r
 
-    def _download_name(self, name, timeout=DEFAULT_TIMEOUT, download_directory=None,
+    def _download_name(self, name, timeout=lbrynet_settings.download_timeout, download_directory=None,
```

Comment:
default argument values should always be immutable.  You'll run into weird behavior otherwise.  This would be better as 
```
def _download_name(self, name, timeout=None, ...):
    if timeout is None:
        timeout = lbrynet_settings.download_timeout
```